docs/packaging_policy.md

```diff
@@ -0,0 +1,18 @@
+# Packaging Policy
+
+This document gathers all the relevant information regarding the general lines to follow while creating new recipes that will eventually be
+part of this repository.
+
+## Sources
+
+- Recipes should always build packages from library sources.
+
+- Library sources should come from an official origin like the library source code repository or the official release/download webpage.
+  Source forks or unofficial origin of sources are not allowed.
+
+- If library sources cannot be downloaded from their official origin or cannot be consumed directly due to their format, an issue should be
+  opened in order to discuss alternative solutions.
+
+- Library sources that are not publicly available will not be allowed in this repository even if the license allows their redistribution.
```

Comment:
IMO these two points are the same: sources that are not accessible. 

We are not storing sources. The recommendation is to contact the publisher and ask them to provide the sources in a way/format that can be consumed programmatically.